src/output.c

```diff
@@ -435,6 +435,7 @@ output_from_wlr_output(struct server *server, struct wlr_output *wlr_output)
 struct wlr_box
 output_usable_area_in_layout_coords(struct output *output)
 {
+	assert(output);
```

Comment:
Alternative:
```c
if (!output) {
    return (struct wlr_box){0};
```

And then remove the new check from `output_usable_area_from_cursor_coords()`

Reply:
Looks good.